layers/+source-control/version-control/packages.el

```diff
@@ -54,7 +54,8 @@
       ;; If you enable global minor mode
       (when (and (eq version-control-diff-tool 'git-gutter)
                  version-control-global-margin)
-        (global-git-gutter-mode t))
+        (run-with-idle-timer 5 nil (lambda ()
```

Comment:
It's true that now people could use different diff tools for different buffers. I don't remember if that's intentional, but it might be. Ok, lets not use `:defer`, current situation is fine.